hack/local-dev-setup.sh

```diff
@@ -23,15 +24,20 @@ HOST_CLUSTER_NAME=${HOST_CLUSTER_NAME:-"kurator-host"}
 MEMBER_CLUSTER_1_NAME=${MEMBER_CLUSTER_1_NAME:-"kurator-member1"}
 MEMBER_CLUSTER_2_NAME=${MEMBER_CLUSTER_2_NAME:-"kurator-member2"}
 HOST_IPADDRESS=${1:-}
+HOST_CONFIG=${HOST_CONFIG:-"host.yaml"}
```

Comment:
prefer to user one ENV control all this
```
kind-configs-with-worker
- host.yaml
- member1.yaml
- member2.yaml
kind-configs
- host.yaml
- member1.yaml
- member2.yaml 
```
add one parameter like `KIND_WITH_WORK`

copy `kind-configs-with-worker/*yaml` to temp dir, otherwise copy  `kind-configs/*yaml`

Reply:
you're right